src/containers/edit-profile/password-security-tab/change-password-modal/ChangePasswordModal.tsx

```diff
@@ -52,15 +51,16 @@ const ChangePasswordModal = () => {
       title: 'titles.confirmTitle',
       check: true
     })
-    if (confirmed) {
-      resetErrors()
+    if (!confirmed) return
+    try {
```

Comment:
And a line between try statement. Also, I'd recommend you not to use if statement without parentheses, because it's unreadable.
```suggestion
    
    try {
```